Exported pdf is 0 bytes

### Describe the bug

I am able to download the PDF file but it is 0 bytes and fails to open.  All other file types download/open correctly.

![files](https://github.com/user-attachments/assets/6b5b0ec4-e5d1-446e-bc37-ab6c933388da)


### Expected behavior

PDF should have content and open as expected.

### Minimal reproducible example

```
var grid = new Grid<String>();
grid.addColumn(String::toString).setHeader("Name");
grid.addColumn(String::toString).setHeader("Email");

grid.setItems("A", "B", "C");

var exporter = GridExporter.createFor(grid);
exporter.setTitle("Grid Export Title");
exporter.setFileName("GridExport_" + new SimpleDateFormat("yyyyddMM").format(Calendar.getInstance().getTime()));
exporter.setAutoAttachExportButtons(true);

add(grid);
```

### Add-on Version

2.3.2

### Vaadin Version

24.4.3

### Additional information

Chrome (latest)

Problem occurs here: https://github.com/FlowingCode/GridExporterAddon/blob/ec5d5ad7bda3d1509a95c3af7a2daac17b3c7521/src/main/java/com/flowingcode/vaadin/addons/gridexporter/PdfInputStreamFactory.java#L62

Here the stacktrace:
```
Exception in thread "Thread-83" java.lang.NoClassDefFoundError: javax/xml/bind/ValidationEventHandler
	at com.flowingcode.vaadin.addons.gridexporter.PdfInputStreamFactory$1.run(PdfInputStreamFactory.java:63)
	at java.base/java.lang.Thread.run(Thread.java:1570)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.ValidationEventHandler
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 2 more
```

--- a/pom.xml
+++ b/pom.xml
@@ -136,23 +136,28 @@
 		</dependency>
 		<dependency>
 		    <groupId>org.docx4j</groupId>
-		    <artifactId>docx4j-JAXB-Internal</artifactId>
-		    <version>8.3.8</version>
-		</dependency>
-		<dependency>
-		    <groupId>org.docx4j</groupId>
 		    <artifactId>docx4j-JAXB-ReferenceImpl</artifactId>
-		    <version>8.3.8</version>
+		    <version>11.5.0</version>
 		</dependency>
 		<dependency>
 		    <groupId>org.docx4j</groupId>
 		    <artifactId>docx4j-JAXB-MOXy</artifactId>
-		    <version>8.3.8</version>
+		    <version>11.5.0</version>
 		</dependency>
 		<dependency>
 		    <groupId>org.docx4j</groupId>
 		    <artifactId>docx4j-export-fo</artifactId>
-		    <version>8.3.8</version>
+		    <version>11.5.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-compress</artifactId>
+			<version>1.27.1</version>
+		</dependency>
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>2.16.1</version>
 		</dependency>
 		<dependency>
 		   <groupId>com.flowingcode.addons</groupId>
@@ -365,7 +370,7 @@
 							<doclint>none</doclint>
 							<failOnWarnings>true</failOnWarnings>
 							<links>
-								<link>https://javadoc.io/doc/com.vaadin/vaadin-platform-javadoc/${vaadin.version}</link>								
+								<link>https://javadoc.io/doc/com.vaadin/vaadin-platform-javadoc/${vaadin.version}</link>
 							</links>
 						</configuration>
 					</plugin>
@@ -527,7 +532,7 @@
 		</profile>
 		<profile>
 			<id>demo-jar</id>
-			<build>		
+			<build>
 				<plugins>
 					<plugin>
 						<groupId>org.apache.maven.plugins</groupId>
@@ -551,7 +556,7 @@
 					</plugin>
 				</plugins>
 			</build>
-		</profile>		
+		</profile>
 	</profiles>
 
 </project>

